app/src/androidTest/java/com/kks/nimblesurveyjetpackcompose/di/FakeRepoModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.kks.nimblesurveyjetpackcompose.di
+
+import com.kks.nimblesurveyjetpackcompose.repo.login.LoginRepo
+import com.kks.nimblesurveyjetpackcompose.ui.presentation.splash.SplashScreenTest
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.components.SingletonComponent
+import dagger.hilt.testing.TestInstallIn
+import javax.inject.Singleton
+
+@Module
+@TestInstallIn(
+    components = [SingletonComponent::class],
+    replaces = [RepositoryModule::class]
+)
+class FakeRepoModule {
+    @Singleton
+    @Provides
+    fun provideFakeLoginRepo(): LoginRepo = SplashScreenTest.FakeLoginRepo()
```

Comment:
How about moving `FakeLoginRepo` out of `SplashScreenTest` to a separate file?

Reply:
Made changes in [Home Screen PR](https://github.com/kaungkhantsoe/NimbleSurveyJetpackCompose/pull/74)